src/messages/messages.json

```diff
@@ -17,6 +18,8 @@
   "converted_sarif_report": "SARIF report generated successfully: {0}",
   "failed_parse_report": "Failed to parse the Parasoft XML Static Analysis report: {0}",
   "parsed_parasoft_static_analysis_report": "Found {0} vulnerabilities for report: {1}",
+  "vulnerability_details_description_limitation": "The detail description for vulnerability '{0}' contains {1} characters. It cannot contain more than 2000 characters, due to the limitation of Bitbucket's API",
```

Comment:
Can we merge actions 21 and 22?

  The detail description for vulnerability '{0}' contains {1} characters. Truncate it to '{2}' characters. Original detail was:\n {3}